src/cargo/ops/cargo_new.rs

```diff
@@ -435,12 +435,20 @@ mod tests {
     Ok(())
 }
 
+fn get_environment_variable(variables: &Vec<&str>)-> Option<String>{
```

Comment:
`&Vec<T>` is typically more idiomatically written as `&[T]` in this case (you can avoid the `vec!` as well)